config/level1_cor.yaml

```diff
@@ -3,9 +3,9 @@ global_attrs:
     location: ${main.location}
     location_coord: "{location_coordinates}"
     campaign_id: ${main.campaign_id}
-    platform: ${main.platform}
-    instrument: "Radiosonde RS41-SGP by Vaisala"
-    instrument_id: "Vaisala-RS"
+    platform: ${level0.platform}
+    instrument: ${level0.instrument}
+    instrument_id: ${level0.instrument_id}
```

Comment:
I would keep this config in the main.yaml and just create a main.yaml for each platform/instrument.